helm/cluster-api-core/values.yaml

```diff
@@ -2,12 +2,10 @@ name: cluster-api-core
 
 images:
   controller:
-    registry: quay.io
     name: giantswarm/cluster-api-controller
-    tag: c39dc00632b621b18df7ec4ec0c73ec6bc6736ec
+    tag: 815685c3d03ac5ca7b6eed6c3a3e3aafd0b96ad9
```

Comment:
Do we want to tag it with a version at some point?

Reply:
Yeah we should.